addons/stock_account/models/product.py

```diff
@@ -172,6 +172,11 @@ def _get_latest_cumulated_value(self, not_move=False):
         domain = [
             ('product_id', '=', self.id),
             ('state', '=', 'done'),
+            ('company_id', '=', self.company_id.id), 
+            '|', '&', ('location_id.usage', 'in', ('internal', 'transit')), 
+            ('location_dest_id.usage', 'not in', ('internal', 'transit')), 
+            '&', ('location_id.usage', 'not in', ('internal', 'transit')), 
+            ('location_dest_id.usage', 'in', ('internal', 'transit'))
```

Comment:
what about inter company transit?
so we should look for location_dest_id.company (inter company transit have no company set on their transit location)